packages/cli/commands/logs.js

```diff
@@ -10,15 +10,16 @@ const { outputLogs } = require('@hubspot/cli-lib/lib/logs');
 const {
   getFunctionLogs,
   getLatestFunctionLog,
-} = require('@hubspot/cli-lib/api/results');
+} = require('@hubspot/local-dev-lib/api/functions');
 const { tailLogs } = require('../lib/serverlessLogs');
 const { loadAndValidateOptions } = require('../lib/validation');
 const { i18n } = require('../lib/lang');
+const { EXIT_CODES } = require('../lib/enums/exitCodes');
 
 const i18nKey = 'cli.commands.logs';
 
 const handleLogsError = (e, accountId, functionPath) => {
-  if (e.statusCode === 404) {
+  if (e.response.status === 404 || e.response.status == 400) {
```

Comment:
You could also use the util [here](https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/errors/apiErrors.ts#L16) (might be overkill though tbh)